presto-tests/src/test/java/io/prestosql/execution/EventsAwaitingQueries.java

```diff
@@ -43,7 +43,13 @@ MaterializedResult runQueryAndWaitForEvents(@Language("SQL") String sql, int num
         return runQueryAndWaitForEvents(sql, numEventsExpected, session, Optional.empty());
     }
 
-    MaterializedResult runQueryAndWaitForEvents(@Language("SQL") String sql, int numEventsExpected, Session session, Optional<String> expectedExceptionRegEx)
+    MaterializedResult runQueryAndWaitForEvents(@Language("SQL") String sql, int numEventsExpected, Session session, String expectedExceptionRegEx)
+            throws Exception
+    {
+        return runQueryAndWaitForEvents(sql, numEventsExpected, session, Optional.of(expectedExceptionRegEx));
+    }
+
+    private MaterializedResult runQueryAndWaitForEvents(@Language("SQL") String sql, int numEventsExpected, Session session, Optional<String> expectedExceptionRegEx)
```

Comment:
> Add simplifying override

- override -> overload ?
- why one is  private and anotehr package private?

Reply:
gone